model/src/main/kotlin/FileFormat.kt

```diff
@@ -83,6 +83,19 @@ fun File.mapper() = FileFormat.forFile(this).mapper
  */
```

Comment:
Commit message nit: The last sentence should be "... these functions *allow* that."